spark-connect/client/src/test/scala-spark-master/io/delta/connect/tables/DeltaTableSuite.scala

```diff
@@ -19,6 +19,8 @@ package io.delta.tables
 import java.io.File
 import java.text.SimpleDateFormat
 
+import org.apache.spark.sql.delta.tables.RemoteSparkSession
```

Comment:
Unrelated change?

Reply:
We changed the package of `RemoteSparkSession`from `io.delta.tables` to ` org.apache.spark.sql.delta.tables` because `SparkConnectClient` is `private[sql]`.

https://github.com/apache/spark/blob/3edc9c23a723a92c5a951cea0436529de65c640a/sql/connect/common/src/main/scala/org/apache/spark/sql/connect/client/SparkConnectClient.scala#L40

So now `DeltaTableSuite` doesn't have the same package name as `RemoteSparkSession`, we have to include explicitly.